pyqtgraph/metaarray/MetaArray.py

```diff
@@ -631,39 +631,45 @@ def prettyInfo(self):
         s = ''
         titles = []
         maxl = 0
-        for i in range(len(self._info)-1):
+        for i in range(len(self._info)):
             ax = self._info[i]
             axs = ''
             if 'name' in ax:
                 axs += '"%s"' % str(ax['name'])
             else:
-                axs += "%d" % i
+                axs += '"%d"' % i
```

Comment:
Adding the quotes here implies that the axis has a string name rather than just the integer index.